var/spack/repos/builtin/packages/ospray/package.py

```diff
@@ -42,37 +43,44 @@ class Ospray(CMakePackage):
     depends_on("rkcommon@1.7:1.9", when="@2.7.0:2.8")
     depends_on("rkcommon@1.9", when="@2.9.0")
     depends_on("rkcommon@1.10:", when="@2.10.0:")
-    depends_on("rkcommon@1.11:", when="@2.11:")
-    depends_on("rkcommon@1.12:", when="@3:")
-    depends_on("rkcommon@1.13:", when="@3.1:")
+    depends_on("rkcommon@1.11.0", when="@2.11:2.12")
+    depends_on("rkcommon@1.12.0", when="@3.0")
+    depends_on("rkcommon@1.13.0", when="@3.1")
+    depends_on("rkcommon@1.14.0", when="@3.2")
     depends_on("embree@3.12: +ispc")
     depends_on("embree@3.13.1:", when="@2.7.0:")
     depends_on("embree@:3", when="@:2.10")
     depends_on("embree@4:", when="@2.11:")
     depends_on("embree@4.3:", when="@3:")
+    depends_on("embree@4.3.3:", when="@3.2:")
     with when("+volumes"):
         depends_on("openvkl@0.13.0:1", when="@2")
         depends_on("openvkl@1.0.1:", when="@2.7.0:")
         depends_on("openvkl@1.2.0:", when="@2.9.0:")
         depends_on("openvkl@1.3.0:", when="@2.10.0:")
         depends_on("openvkl@1.3.2:", when="@2.11:2")
         depends_on("openvkl@2:", when="@3:")
+        depends_on("openvkl@2.0.1:", when="@3.2:")
     with when("+denoiser"):
         depends_on("openimagedenoise@1.2.3:")
         depends_on("openimagedenoise@1.3:", when="@2.5:")
         depends_on("openimagedenoise@:1", when="@:2.11")
         depends_on("openimagedenoise@2:", when="@2.12:")
         depends_on("openimagedenoise@2.1:", when="@3:")
+        depends_on("openimagedenoise@2.3:", when="@3.2:")
     depends_on("ispc@1.14.1:", type=("build"))
     depends_on("ispc@1.16.0:", when="@2.7.0:", type=("build"))
     depends_on("ispc@1.18.0:", when="@2.10.0:", type=("build"))
     depends_on("ispc@1.19.0:", when="@2.11.0:", type=("build"))
     depends_on("ispc@1.20.0:", when="@2.12.0:", type=("build"))
     depends_on("ispc@1.21.1:", when="@3:", type=("build"))
+    depends_on("ispc@1.23.0:", when="@3.2:", type=("build"))
     depends_on("tbb")
 
-    depends_on("mpi", when="+mpi")
-    depends_on("snappy@1.1.8:", when="+mpi")
+    with when("+mpi"):
+        depends_on("mpi")
+        depends_on("snappy@1.1.8:")
+        depends_on("snappy@1.2.1:", when="@3.2:")
```

Comment:
Quickly looking into ospray's cmake files didn't reveal to me why snappy@1.2.1 is required now. I only could find that 1.2.1 is used in ospray's super build script, but starting with version 3.1. Perhaps, this can be removed?

Reply:
I indeed based this on the super build https://github.com/RenderKit/ospray/commit/6ad92f0deed31ac2325e51ef1e2593e443aa484e which seems to apply for 3.2, but not 3.1. There is no other version requirement for snappy anywhere. I didn't attempt to build with the older snappy. I based this change on the use of 'bump dependencies' in the commit, though I could imagine it is not a requirement. I'll attempt to rebuild with the older snappy to verify that works.

Reply:
@wdconinc / @aumuell: Indeed, built this after rm / `# depends_on("snappy@1.2.1:", when="@3.2:")`:
```py
bin/spack install -v ospray+mpi@3.2.0 ^snappy@1.1.8 # (and @1.1.9, @1.1.10)
[+] /home/bkaindl/spack/opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/snappy-1.1.8-==> ospray: Successfully installed ospray-3.2.0-lb4ryuoi5mwpbagqvkh7xz7xb372fu67
```
(no other testing besides that)

Reply:
Am I to resolve this conversation? I don't know how...

Reply:
> @bernhardkaindl I don't feel like a natural choice as a reviewer, as I'm not really a user of Ospray. I don't really know how my name sneaked into the comment in this package.

@aumuell, I think you would be welcome to review pull requests, as I think that `spack` is (or we are) in need of more helping hands joining us in review to approve or improve PRs by reviewing them. I guess that Todd Gamblin (as the leader) and the others would appreciate more help.

> Am I to resolve this conversation? I don't know how...

In principle, you should see the button "Resolve conversation" below the conversation when your browser tab is logged into your accrount and re-loaded. But I am not an expert if it is missing. Todd could give you that permission if you don't have it. 

Even though I'm not affiliated with the GitHub actions/cache org/repo, I could still resolve this (oudated) conversation: https://github.com/actions/cache/pull/1326#discussion_r1590079425 Do you see the button the resolve that discussion?

Reply:
And, yes, we need to resove all discussions in PRs before the settings on GitHub for spack allow the PR to be merged.

But I can mark it as resolved now.

Reply:
Yes, please do: I cannot find such a button on either conversation